on_device_training/mobilevit/README.md

```diff
@@ -0,0 +1,47 @@
+# MobileVIT End-to-End Example for On-Device Training
+This folder contains an end-to-end example for on-device training in a C# console app using the ONNXRuntime Training libraries.
+
+On-device training contains two steps: an "offline" pre-processing step that happens off of the device in question, and the training step that happens on the device. 
```

Comment:
This sentence is not needed... this information should be captured in the highlevel readme. You may want to just link that read me here.